src/data_collector/indicator_pipeline/indicator_pipeline.py

```
@@ -28,6 +28,131 @@
+                                "date": date_idx.date(),
+                                "feature_category": classify_feature_name(feature_name),
+                                "feature_name": feature_name,
+                                "feature_value": feature_value,
+                                "quality_score": quality_score_val,
+                            }
+                        )
+
+                saved_count = bulk_upsert_technical_features(
+                    rows, page_size=1000, overwrite=config_dict.get("overwrite_existing", True)
+                )
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)